pyfr/integrators/base.py

```diff
@@ -227,3 +130,54 @@ def cfgmeta(self):
             return ret
         else:
             return {'config': cfg, 'config-0': cfg}
+
+
+class BaseCommon(object):
+    def _init_reg_banks(self, backend):
```

Comment:
Do we need a backend argument here?

Reply:
No actually we don't. Amending this to the previous commit.